Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -27,10 +29,27 @@ public class BoundaryVisualizationDemo : MonoBehaviour, IMixedRealityBoundaryHan
         [SerializeField]
         private bool showPlayArea = true;
 
+        [SerializeField]
+        private bool showTrackedArea = true;
+
+        [SerializeField]
+        private bool showBoundaryWalls = true;
+
+        [SerializeField]
+        private bool showBoundaryCeiling = true;
+
         #region MonoBehaviour Implementation
 
+        private void Awake()
+        {
+            markerParent = new GameObject();
```

Comment:
Why not just put this in the scene since it's for the demo?

Reply:
parenting this way for teleportation support.